configure.ac

```diff
@@ -293,7 +293,7 @@ case "$with_platform" in
 	AC_SUBST(XTENSA_CORE)
 
 	AC_DEFINE([CONFIG_APOLLOLAKE], [1], [Configure for Apollolake])
-	AC_DEFINE([CONFIG_BOOT_LOADER], [1], [Configure Boot Loader])
+	AC_DEFINE([CONFIG_KABYLAKE], [1], [Configure for Kabylake])
```

Comment:
KABYLAKE is based on APOLLOLAKE? That seems very counter-intuitive given the boot flow and memory architecture differences?

Reply:
There are some differences in memory map, ROM, bootloading and manifest but otherwise HW IP is similar. I will be rebasing this after @jajanusz has merged his new build system.